statistics/selectivity.go

```diff
@@ -192,9 +192,9 @@ func (coll *HistColl) Selectivity(ctx sessionctx.Context, exprs []expression.Exp
 			if err != nil {
 				return 0, nil, errors.Trace(err)
 			}
-			nodes = append(nodes, &StatsNode{Tp: colType, ID: id, mask: maskCovered, Ranges: ranges, numCols: 1})
-			if colInfo.isHandle {
-				nodes[len(nodes)-1].Tp = pkType
+			nodes = append(nodes, &StatsNode{Tp: ColType, ID: id, mask: maskCovered, Ranges: ranges, numCols: 1})
+			if colInfo.IsHandle {
```

Comment:
Hi @winoros 

Could you please explain the meaning of `IsHandle`?  

Does that mean colInfo with `true` value is the primary key?

Thanks